doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1331,34 +568,4 @@ cJSON
 Documentation
 =============
 
-* Added:
-
-  * The :ref:`ug_app_dev` section, which includes pages from the :ref:`configuration_and_build` section and from the removed Device configuration guides section.
-  * The :ref:`dfu_tools_mcumgr_cli` page after it was removed from the Zephyr repository.
-  * The :ref:`ug_nrf54h20_suit_soc_binaries` page.
-  * The :ref:`ug_nrf54h20_suit_push` page documenting the SUIT push model-based update process.
-  * The :ref:`ug_nrf54h20_suit_recovery` page.
-  * The :ref:`nrf_rpc_uart` page.
-  * The :ref:`ug_bt_stack_arch` and the :ref:`ug_bt_solution` documentation to the :ref:`Bluetooth protocols <ug_bt>` page.
-
-* Updated:
-
-  * The :ref:`ug_nrf70_developing_debugging` page with the new snippets added for the nRF70 driver debug and WPA supplicant debug logs.
-  * The :ref:`programming_params` section on the :ref:`programming` page with information about readback protection moved from the :ref:`ug_nrf5340_building` page.
-  * The :ref:`security` page with a table that provides an overview of the available general security features.
-    This table replaces the subpage that was previously describing these features in more detail and was duplicating information available in other sections.
-  * Restructured the :ref:`app_bootloaders` documentation and combined the DFU and bootloader articles.
-    Additionally, created a new bootloader :ref:`bootloader_quick_start`.
-  * Separated the instructions about building from :ref:`configure_application` and moved it to a standalone :ref:`building` page.
-  * Restructured the :ref:`ug_bt_mesh` documentation for clearer distinction between concepts or overview topics and how-to topics, thus moved some information from the Bluetooth Mesh library sections.
-  * The :ref:`nrf_security_drivers_cracen` section with a reference to the :ref:`ug_nrf54l_cryptography` page.
-  * The :ref:`ug_tfm` page with the correct list of samples demonstrating TF-M.
-  * The :ref:`app_approtect_ncs` section on the :ref:`app_approtect` page with details on setting the Kconfig options and register values to enable AP-Protect.
-
-* Removed:
-
-  * The Device configuration guides section and moved its contents to :ref:`ug_app_dev`.
-  * The Advanced building procedures page and moved its contents to the :ref:`building` page.
-  * nRF70 Series support is upstreamed to Zephyr, hence the documentation is removed from the |NCS|.
```

Comment:
this can be removed now, we have reinstated the docs. But the `nRF70 driver doxygen APIs are still not linked to the documentation`, so, we can change the scope to limit to that.

Reply:
We have a mention about Wi-Fi driver here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-18192/nrf/releases_and_maturity/releases/release-notes-2.8.0.html#wi-fi-drivers. Do we need to add doc-related information since now drivers docs are in NCS, to make it clear?and provide link to the documentation.